source/jormungandr/jormungandr/parking_space_availability/bss/atos.py

```diff
@@ -37,7 +38,7 @@
 
 class AtosProvider(BssProvider):
 
-    OPERATOR = 'Keolis'
+    OPERATORS = ('keolis', 'effia', 'effia transport', u'kéolis')
```

Comment:
it's really minor, but if this list is meant to grow, I think it's better to use a set
=> `OPERATORS = {'keolis', 'effia', 'effia transport', u'kéolis'}`